apache-rat-core/src/main/java/org/apache/rat/ReportConfiguration.java

```diff
@@ -766,21 +704,17 @@ public NoCloseOutputStream(final OutputStream delegate) {
         }
 
         @Override
-        public void write(final int arg0) throws IOException {
+        public void write(int arg0) throws IOException {
             delegate.write(arg0);
         }
 
-        /**
```

Comment:
This javadoc seems helpful to me as I do not have to have a look into the source in order to see what is special about this class ..... I'd want to keep it here.